samples/TestApplicationShared/App.cs

```diff
@@ -18,35 +18,35 @@ public class App : Application
     {
         public App()
         {
-			// TODO: I believe this has to happen before we select sub systems. Can we
-			// move this safely into Application itself? Is there anything in here
-			// that is platform specific??
-			//
+            // TODO: I believe this has to happen before we select sub systems. Can we
+            // move this safely into Application itself? Is there anything in here
+            // that is platform specific??
+            //
```

Comment:
`RegisterServices` is virtual, and it is dangerous to call virtual methods from base constructors, which is why it's called explicitly in the `App` constructor: http://stackoverflow.com/a/119543